### PR TITLE
Add ability to check for a license

### PR DESCRIPTION
**Pull request by jletey**
@nikitavoloboev Here's a rough draft ... it works fine for me

Also fixed another indentation error

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+	"io/ioutil"
+	"log"
 	"os"
 
 	kingpin "gopkg.in/alecthomas/kingpin.v2"
 )
 
 var (
 	app         = kingpin.New("license-up", "A command-line tool to make licences.")
+	force       = app.Flag("force", "Force to create a license.").Short('f').Bool()
```
Comment (nikivdev) [inline main.go]: Rewrite to `Create a license even if license already exists.`

File: main.go [hunk 2]
```diff
@@ -25,6 +30,30 @@ var (
...
+	if bool(*force) == false {
+		files, err := ioutil.ReadDir(".")
+		if err != nil {
+			log.Fatal(err)
+		}
+		reader := bufio.NewReader(os.Stdin)
+		overwrite := false
+		for _, f := range files {
+			if f.Name() == "LICENSE" {
+				fmt.Print("There is already a license present in current directory. Do you want to overwrite it with a new one? [y/N] ")
```
Comment (nikivdev) [inline main.go:45]: Right now, if there is no `LICENSE` at all and you run say `license-up mit Nikita Voloboev nikitavoloboev.xyz`. No license will get created.

Comment (nikivdev) [inline main.go:45]: It would be nice to have versions added to the tool so you can run `license-up --version` and see what version you are on.

Comment (jletey) [reply main.go:45]: I'll make a separate pull request for this ... btw, what version number are we on? 1.0.0?

Comment (nikivdev) [reply main.go:45]: We can do 1.0.0 release yeah.

Comment (jletey) [reply main.go:45]: cool ... I'll make a PR right now

**Pull request merged**